src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search__bar search__bar--big"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <label class="search__label search__label--big" for="big"></label>
+      <input
+        class="search__input search__input--big"
```

Comment:
```suggestion
      class="search__bar"
      action="#"
      method="get"
      data-qa="big"
    >
      <label class="search__label" for="big"></label>
      <input
        class="search__input"
```
it is redundant to create modifiers for big form.
set all styles for element class and update just necessary properties in --small modifiers